languages/_sidebar.md

```diff
@@ -4,16 +4,28 @@
   - [Bash](/languages/bash/README.md)
   - [C](/languages/c/README.md)
   - [C#](/languages/csharp/README.md)
+  - [Clojure](/languages/clojure/README.md)
+  - [Common Lisp](/languages/common-list/README.md)
   - [Dart](/languages/dart/README.md)
+  - [Elixir](/languages/elixir/README.md)
+  - [Emacs LISP](/languages/emacs-lisp/README.md)
```

Comment:
```suggestion
  - [Emacs Lisp](/languages/emacs-lisp/README.md)
```

Reply:
I'll fix this in a separate PR, as there are multiple places this has to be changed.

Reply:
I noticed that [exercism/elisp's config.json](https://github.com/exercism/elisp/blob/master/config.json) uses "Emacs Lisp", which stylistically coincides with the v3 "Common Lisp" track, so this is probably the way.

Reply:
See https://github.com/exercism/v3/pull/151/files